### PR TITLE
[plugin.video.cbc-sports] 1.0.1

### PR DESCRIPTION
**Pull request by jbinkley60**
Description

Initial new Kodi Matrix addon plugin.video.cbc-sports submission.

Get the latest headlines, opinion, and videos from CBC Sports.  Watch live events, replays, and highlights.  Some content blocked outside of Canada.

Checklist:

[X ] My code follows the add-on rules and piracy stance of this project.
[X ] I have read the CONTRIBUTING document
[X ] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0
[X ] My code follows the add-on rules and piracy stance of this project.
[X ] I have read the CONTRIBUTING document
Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :

Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
add-on development wiki page.
Kodi pydocs provide information about the Python API
PEP8 codingstyle which is considered best practise but not mandatory.
This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at Codacy. You can create your own account as well to continuously monitor your python coding before submitting to repo.
Development questions can be asked in the add-on development section on the Kodi forum.

### CODE REVIEW

File: plugin.video.cbc-sports/addon.py [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+            return
+
+	if errfound > -1:
+		xbmcgui.Dialog().notification(name, translation(30010), defaultimage, notetime, False)
+		return
+	if cbclog >= 1:
+	    xbmc.log('CBC Sports Live Schedule stream return data: ' + sdata, xbmc.LOGINFO)
+
+	try:	
+	    xbmc.Player().play( stream, listitem )
```
Comment (enen92) [inline plugin.video.cbc-sports/addon.py]: Sorry, Player().play() is to be used by scripts not plugins. Please adjust this to use xbmcplugin.setResolvedUrl

File: plugin.video.cbc-sports/addon.py [hunk 2]
```diff
@@ -0,0 +1,417 @@
...
+	    return
+	stream = (re.compile('src="(.+?)"').findall(str(contents))[0])
+	if mp4base not in stream:
+		#stream = mp4base + stream
+		stream = stream
+	listitem = xbmcgui.ListItem(name)
+	listitem.setArt({'thumb': defaultimage, 'icon': defaultimage})
+	if cbclog >= 1:
+	    xbmc.log('CBC Sports Live Schedule Playback stream: ' + str(stream), xbmc.LOGINFO)
+	xbmc.Player().play( stream, listitem )
```
Comment (enen92) [inline plugin.video.cbc-sports/addon.py]: Same

File: plugin.video.cbc-sports/addon.py [hunk 3]
```diff
@@ -0,0 +1,417 @@
...
+	liz = xbmcgui.ListItem(name)
+	plot_text = ''
+	if plot:
+	    plot_text = plot
+	else:
+	    plot_text = name
+	aired_text = ''
+	if aired:
+            aired_text = aired
+	infoLabels={ "Title": name,
```
Comment (enen92) [inline plugin.video.cbc-sports/addon.py]: This method of setting infolabels is deprecated in v20

File: plugin.video.cbc-sports/addon.xml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    <import addon="script.module.html5lib" version="0.999.0"/> 
+    <import addon="script.module.requests" version="2.9.1"/> 
+    </requires>
+    <extension point="xbmc.python.pluginsource"
+               library="addon.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <platform>all</platform>
+        <license>GPL 2.0 or Later</license>
```
Comment (enen92) [inline plugin.video.cbc-sports/addon.xml]: GPL-2.0-or-later

File: plugin.video.cbc-sports/addon.xml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+        <assets>
+        <icon>/resources/icon.png</icon>
+        <fanart>/resources/fanart.jpg</fanart>
+        </assets>
+        <summary lang="en_GB">Live Events, Replays, and Highlights from CBC Sports</summary>
+        <description lang="en_GB">Get the latest headlines, opinion, and video from CBC Sports.  Watch live events, replays, and highlights.  Region blocked for Canada only.</description>
+        <disclaimer lang="en_GB"></disclaimer>
```
Comment (enen92) [inline plugin.video.cbc-sports/addon.xml]: Remove if empty

File: plugin.video.cbc-sports/addon.xml [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <platform>all</platform>
+        <license>GPL 2.0 or Later</license>
+        <assets>
+        <icon>/resources/icon.png</icon>
+        <fanart>/resources/fanart.jpg</fanart>
+        </assets>
+        <summary lang="en_GB">Live Events, Replays, and Highlights from CBC Sports</summary>
```
Comment (enen92) [inline plugin.video.cbc-sports/addon.xml:22]: Addon.xml missed source tag to point to your upstream repo

**Review by enen92 — CHANGES_REQUESTED** [review]
Aside from the requested changes please use xbmc.LOGDEBUG

File: plugin.video.cbc-sports/addon.py (hunk 1 above)
Comment (jbinkley60) [reply plugin.video.cbc-sports/addon.py]: Code has been updated to use xbmcplugin.setResolvedUrl

File: plugin.video.cbc-sports/addon.py (hunk 2 above)
Comment (jbinkley60) [reply plugin.video.cbc-sports/addon.py]: Code has been updated to use xbmcplugin.setResolvedUrl

File: plugin.video.cbc-sports/addon.py (hunk 3 above)
Comment (jbinkley60) [reply plugin.video.cbc-sports/addon.py]: Noted.  Will modify once v20 is published.

File: plugin.video.cbc-sports/addon.xml (hunk 4 above)
Comment (jbinkley60) [reply plugin.video.cbc-sports/addon.xml]: Fixed in addon.xml file

File: plugin.video.cbc-sports/addon.xml (hunk 5 above)
Comment (jbinkley60) [reply plugin.video.cbc-sports/addon.xml]: Fixed in addon.xml file

File: plugin.video.cbc-sports/addon.xml (hunk 6 above)
Comment (jbinkley60) [reply plugin.video.cbc-sports/addon.xml:22]: Fixed in addon.xml file

**Comment by jbinkley60** [comment]
All requested changes have been made.  Please review again.

**Comment by jbinkley60** [comment]
> Aside from the requested changes please use xbmc.LOGDEBUG

I split minor logging and detailed logging between LOGINFO and LOGDEBUG.

**Comment by jbinkley60** [comment]
Closing PR due to lack of response.  Will resubmit with newer version of addon.

**Pull request closed without merging**